api/api.proto

```diff
@@ -533,9 +533,13 @@ message SdkAlertsResourceIdQuery {
 // Each query object is one of the three query types and a list of
```

Comment:
You are missing changing the version to 0.11.0 and updating the SDK_CHANGELOG.md